### ISSUE TITLE
Brew Deprecation Warning: Calling bottle :unneeded is deprecated!

### ISSUE THREAD

**Issue by d3an** [weaveworks/eksctl#4458]
**What were you trying to accomplish?**
When I run `brew install weaveworks/tap/eksctl` it gives this warning:
```commandline
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the weaveworks/tap tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/weaveworks/homebrew-tap/Formula/eksctl.rb:9
```
Basically, is there any way to fix/stop it from being issued in the future?

**What happened?**
I already have `eksctl` installed, but whenever I update it I get the following warning message via brew:
```commandline
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the weaveworks/tap tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/weaveworks/homebrew-tap/Formula/eksctl.rb:9
```

**How to reproduce it?**
```commandline
$ brew tap weaveworks/tap
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 2 formulae.
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the weaveworks/tap tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/weaveworks/homebrew-tap/Formula/eksctl.rb:9

$ brew install weaveworks/tap/eksctl

Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the weaveworks/tap tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/weaveworks/homebrew-tap/Formula/eksctl.rb:9

Warning: weaveworks/tap/eksctl 0.73.0 is already installed and up-to-date.
To reinstall 0.73.0, run:
  brew reinstall eksctl
$
```

**Logs**

**Anything else we need to know?**
I'm running MacOS 10.15.7 with zsh.

**Versions**

```
$ eksctl info
eksctl version: 0.73.0
kubectl version: v1.22.3
OS: darwin
```

**Comment by Himangini** [comment]
@d3an thanks for reporting this issue. We are working on it currently and will release a fix soon 👍🏻

**Comment by nikimanoledaki** [comment]
We use `goreleaser` to generate the brew tap formula and they removed the deprecated `bottle :unneeded` in their latest release so bumping it from v0.184.0 to v1.0.0 should fix this issue (see goreleaser/goreleaser#2591). 

We'll do a release first thing tomorrow. 

Thank you for raising this! :)
